### ISSUE TITLE
Metadata: don't refresh periodically by default.

### ISSUE THREAD

**Issue by Lorak-mmk** [scylladb/scylla-rust-driver#1008]
Currently driver refreshes metadata in 4 situations:
1. When requested by user
2. When receiving cql event that signals the need to refresh
3. When re-establishing control connection
4. Periodically, by default once every 60s. This can't be disabled (you can practically disable it by setting very long period).

Metadata may be big and so fetching it may take a [lot of resources](https://github.com/scylladb/scylla-rust-driver/issues/786). This will be partially mitigated by https://github.com/scylladb/scylla-rust-driver/issues/595 because only necessary metadata will be fetched by default, but
1. Users who want to download full metadata will still need to pay the full cost
2. Even this limited metadata may be considered too expensive if downloaded unnecesarily

If cql-events are working correctly there should be no need to periodically refresh metadata. I propose to assume that they are working properly, make it possible to fully disable periodical fetching and disable it by default.

**Comment by Lorak-mmk** [comment]
Something I was unaware of: https://github.com/scylladb/scylla-rust-driver/issues/546
Currently driver doesn't refresh on schema change event, only on topology change event.
This needs to be fixed before we can disable periodical fetching.

**Comment by Lorak-mmk** [comment]
Closing in favor of https://github.com/scylladb/scylla-rust-driver/issues/1280
